### ISSUE TITLE
Include support for plotting numeric parameters on the x-axis

### ISSUE THREAD

**Issue by bhwein** [TASBE/TASBEFlowAnalytics#54]
Create fits, determine EC50, hill coefficient etc.

**Comment by bhwein** [comment]
Would you be able to incorporate a plot where you can pick a particular value of CFP MEFL, and then plot plus and minus OFP/IFP MEFL, fold activation, SNR, dSNR etc on the y-axis and parameter value (e.g. time) on the x-axis?

**Comment by jakebeal** [comment]
If I'm understanding correctly, you mean to override the default "active CFPs" range with a fixed CFP range? That is a feature that could (and should) be added --- I have needed it in the past as well.

**Comment by bhwein** [comment]
Not exactly. I think you could already do that with `bins = BinSequence(4,0.1,10,'log_bins');` and set the range? Ideally, I would like to see three parameters at the same time: CFP MEFL, the parameter value, and one of the following OFP/IFP MEFL, fold activation, SNR, dSNR. However, this would need to be a 3D plot, so I was thinking it would be best to collapse the CFP MEFL as a single point that you could choose (e.g., 5x10^7 CFP MEFL) and then look at the other two values. It would also be nice to compare multiple devices on the same plot (i.e. to compare dynamics, compare how changing plasmid amounts changed each device).
